lib/child_process.js

```diff
@@ -496,6 +496,12 @@ function normalizeSpawnArguments(file, args, options) {
   var env = options.env || process.env;
   var envPairs = [];
 
+  // process.env.NODE_V8_COVERAGE always propagates, making it possible to
+  // collect coverage for programs that spawn with white-listed environment.
+  if (process.env.NODE_V8_COVERAGE) {
```

Comment:
is there a way to stop this from happening?

Reply:
there is not at this time? any recommendations.

 It's almost always the case that you will be setting `NODE_V8_COVERAGE` in the context of running a test suite, and that you'd probably like to er on the side of the variable propagating, even though it's pretty common in tests to selectively set `env`, e.g., maybe your test is exercising a specific environment.

Reply:
I'm sure it's quite common, but we should support both sides. perhaps checking if the property already exists (so you can set it to an empty string) would work.

Reply:
I think that's a smart idea, I was thinking even for our own test suite, we might want to be able to remove the variable.